cli/src/commands/init/index.ts

```diff
@@ -161,9 +162,20 @@ export default class Init extends BaseCommand {
     await this.updateConfig();
   }
 
-  async writeEnvFile() {
+  async writeEnvFile(workspace: string, database: string) {
     // TODO: generate a database-scoped API key
-    const apiKey = (await getProfile())?.apiKey;
+    let apiKey = (await getProfile())?.apiKey;
+
+    if (!apiKey) {
+      apiKey = await createAPIKeyThroughWebUI();
+    }
+
+    const fallbackBranch = await this.getBranch(workspace, database, {
+      allowEmpty: true,
+      allowCreate: true,
+      title:
+        'Choose a default development branch. This will be used when you are in a git branch that does not have a corresponding Xata branch (a branch with the same name, or linked explicitely)'
```

Comment:
Yeah, maybe, but I preferred to be very explicit and informational :) Otherwise I think new users won't understand why they need to specify a fallback branch.